src/sentry/api/serializers/models/userrollback.py

```diff
@@ -0,0 +1,41 @@
+from typing import TypedDict
+
+from sentry.api.serializers import Serializer
+
+
+class RollbackOrganizationSerializerResponse(TypedDict):
+    id: int
+    name: str
+    slug: str
+
+
+class RollbackUserSerializerResponse(TypedDict):
+    id: int
+    name: str
+
+
+class RollbackSerializerResponse(TypedDict):
+    organization: RollbackOrganizationSerializerResponse
+    user: RollbackUserSerializerResponse
+    data: dict  # JSON Blob
```

Comment:
is it worth typing data? or are we still determining what will be in that

Reply:
Still unsure what exactly this will look like

Reply:
Yeah the data shape is still to be determined and won't be manipulated by the backend anyway. But it would be good to come back and type it once we've solidified that